### PR TITLE
fix private run a bit

### PR DESCRIPTION
**Pull request by samogot**
You can more easily connect to private server and you don't have to install screeps server globally

### CODE REVIEW

**Comment by Nevrdid** [comment]
will this not break circlecli test ?

File: .screepsrc.example [hunk 1]
```diff
@@ -0,0 +1,7 @@
+;If you launch the server without running the local Steam client, 
+;then the Steam Web API key is required for authenticating users. 
+;It can be obtained on this page: http://steamcommunity.com/dev/apikey
+steam_api_key = SOMEKEY
+
+;The server password which should be provided on user sign in.
+password = somepassword
```
Comment (TooAngel) [inline .screepsrc.example]: Maybe add, that this file needs to be copied to the `test-private-server` directory

**Comment by democratic-collaboration** [comment]
A new review, yeah.

    Votes: 301/505
    Coefficient: 0.59603960396
    Merging in 0 days 0 hours
    Age 4 days 0 hours

**Comment by democratic-collaboration** [comment]
A new review, yeah.

    Votes: 302/506
    Coefficient: 0.596837944664
    Merging in 7 days 1 hours
    Age 1 days 0 hours

**Comment by samogot** [comment]
thanks. I haven't noticed it)

**Pull request merged**